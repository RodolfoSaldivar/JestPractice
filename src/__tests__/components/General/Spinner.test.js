import React from 'react';
import { shallow } from 'enzyme';
import Spinner from '../../../components/General/Spinner';

describe('<Spinner />', () => {
	const spinner = shallow(<Spinner />);

	it('Renders correctly', () => {
		expect(spinner).toMatchSnapshot();
	});

	it('Has correct classes', () => {
		const div = spinner.find({ className: 'lds-ring' });
		expect(div).toHaveLength(1);
	});
});
