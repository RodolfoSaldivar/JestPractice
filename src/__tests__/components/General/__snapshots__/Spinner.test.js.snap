// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> Renders correctly 1`] = `
<div
  className="center"
>
  <div
    className="lds-ring"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
